cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java

```diff
@@ -262,8 +267,6 @@ else if (isDefaultModulesDirSet) {
 			if (m != null) {
 				m.setAccessible(false);
 			}
-
-			thread.setContextClassLoader(oldContextClassLoader);
```

Comment:
hey @Seiphon  why do we not need to re-set the thread context classloader to the old classloader?  Does this not break the classloader afterwards?

Reply:
Actually, when execute method finished , In here https://github.com/liferay/liferay-blade-cli/blob/5108d3b0c897950f0a06181ffcc50f60cbacc949/cli/src/main/java/com/liferay/blade/cli/BladeCLI.java#L1328
,the classloader will be re-set to the old one, so I think we don't need to reset it twice.